client/src/js/components/bhStockOut/bhStockOut.js

```diff
@@ -49,8 +48,8 @@ function bhStockOutController(Depots, moment, Notify, $filter, $q) {
       .then(([inventories, depot]) => {
 
         inventories.forEach(inventory => {
-          inventory.stock_out_date_raw = $date(inventory.stock_out_date);
-          inventory.stock_out_date_parsed = moment(inventory.stock_out_date).fromNow();
+          inventory.stock_out_date_raw = $date(inventory.date);
```

Comment:
To address a bug - the server sends back the `date` now, not the `stock_out_date`.  This is a bug in master, and I'm fixing it here, since I had to investigate it to see if it was me that broke it.